yaml/boundaries.yaml

```diff
@@ -1,75 +1,79 @@
 filters:
+
+  # osm
   - filter:
       admin_level: ['2', '4']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 8
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: '3'}
```

Comment:
2 or 4 for the above, and 6 or 8 for the below, although I couldn't find any actual data that matched the below.